src/mca/state/base/state_base_fns.c

```diff
@@ -397,37 +397,6 @@ void prrte_state_base_print_proc_state_machine(void)
     }
 }
 
-static void cleanup_node(prrte_proc_t *proc)
```

Comment:
This was being called after every proc exited - something I don't think seems right. It'll get cleaned up `check_complete` in state_dvm.c